app/models/header_lookup.rb

```diff
@@ -28,7 +28,7 @@ def pid_lookup_by_scheme(term="", scheme="")
 
   # return PID for provided mesh_header using SPARQL query
   def mesh_term_pid_lookup(mesh_term="")
-    hits = perform_and_parse_mesh_query(mesh_term)
+    hits = perform_and_parse_mesh_query(CGI.escape(mesh_term))
```

Comment:
This is the fix itself. `Obesity-- prevention & control` does not actually return anything, but at least now it can be searched without breaking things.